### PR TITLE
Added request timeout instance variable to API.

### PR DESCRIPTION
**Pull request by mharista**
Allows user to specify client get/post timeouts used by API calls. Issue #7

### CODE REVIEW

**Comment by mharista** [comment]
I had a discussion with Cheyne about this change and he mentioned doing this in a way where the new timeout is a part of the CvpClient instead of CvpApi. This would require less code changes and keep the CvpApi class agnostic to any timeout/connection parameters. A quick usage example is below:

clnt = CvpClient()
clnt.connect(....)
clnt.response_timeout = 40
clnt.api.get_cvp_info()
clnt.response_timeout = 30

I decided to add the functionality to CvpApi because it seems more intuitive to me at have a timeout for api calls accessed via the api. It also makes use of the get and post client functions already accepting a timeout parameter. Same code with one difference is below:

clnt = CvpClient()
clnt.connect(....)
clnt.api.response_timeout = 40
clnt.api.get_cvp_info()
clnt.api.response_timeout = 30

I don't have a preference either way, but I'm curious about others preferences.

**Comment by devrobo** [comment]
You are right, in looking at the example code it is more intuitive to handle the timeout in the api code.

File: cvprac/cvp_api.py [hunk 1]
```diff
@@ -61,22 +61,23 @@ class CvpApi(object):
...
+        self.request_timeout = request_timeout
 
     def get_cvp_info(self):
         ''' Returns information about CVP.
 
             Returns:
                 cvp_info (dict): CVP Information
         '''
-        return self.clnt.get('/cvpInfo/getCvpInfo.do')
+        return self.clnt.get('/cvpInfo/getCvpInfo.do', self.request_timeout)
```
Comment (devrobo) [inline cvprac/cvp_api.py]: The timeout parameter is a named parameter so I think the syntax should be:
timeout=self.request_timeout

Comment (devrobo) [inline cvprac/cvp_api.py]: The timeout parameter is a named parameter so I think the syntax should be:
timeout=self.request_timeout . I think this should be done for all calls below.

**Pull request merged**